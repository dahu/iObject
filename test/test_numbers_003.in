" test iObject Numbers

-12345
-12345
+12345
+12345
-12345 -12345
-12345 -12345
-12345ax
ax-12345
ax-12345ax
ax -12345 ax

