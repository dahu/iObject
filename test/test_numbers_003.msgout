
"test_numbers_003.in" 
"test_numbers_003.in" 13L, 123C
Warning: terminal cannot highlight












"test_numbers_003.out" 
"test_numbers_003.out" [New] 13L, 69C written
