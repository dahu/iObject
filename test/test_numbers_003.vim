source ../plugin/iObject.vim
edit test_numbers_003.in

" delete unsigned numbers
1
" move to leading -/+ sign character
call search('-', 'W')
normal cini
call search('-', 'W')
normal canj
call search('+', 'W')
normal cini
call search('+', 'W')
normal canj

" move to start of numbers
call search('\d', 'W')
normal canj
call search('\d', 'W')
normal cank
call search('\d', 'W')
normal cinj
call search('\d', 'W')
normal cink

" move inside a number or to the end of a number
call search('\d\d', 'We')
normal cinj
call search('\d\+', 'We')
normal cank
call search('\d\d', 'We')
normal canj
call search('\d\+', 'We')
normal cink

write test_numbers_003.out
quit!
