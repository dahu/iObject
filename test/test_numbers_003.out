" test iObject Numbers

-i
j
+i
j
j k
-j -k
-jax
axk
axjax
ax -k ax

